packages/apsara-ui/src/Listing/Listing.types.tsx

```diff
@@ -1,28 +1,26 @@
 import { IVirtualTable } from "../Table/VirtualisedTable";
-export interface ListingProps {
-    list?: any[];
+import { Column } from "../TableV2/VirtualisedTable";
+export interface ListingProps<T> {
+    list?: T[];
     loading?: boolean;
     resourceName?: string;
     resourcePath?: string;
     rowKey?: string;
     className?: string;
     tableProps?: {
-        getColumnList?: any;
-        handleRowClick?: (event: any, rowIndexData: any) => void;
+        getColumnList?: (path: string, sortedInfo: any) => Column<T>[];
         selectedRowId?: number;
         scroll?: any;
     } & Omit<IVirtualTable, "columns" | "items">;
     filterProps?: { filterFieldList?: IGroupOptions[] };
     searchProps?: {
         searchPlaceholder?: string;
-        searchFields?: any[];
+        searchFields?: string[];
         disabled?: boolean;
     };
-    renderExtraFilters?: any;
-    renderHeader?: any;
-    renderBody?: any;
-    calculateRowHeight?: any;
-    calculateColumnWidth?: any;
+    renderExtraFilters?: React.ReactNode | null;
```

Comment:
I believe `React.ReactNode` is already covering `null`. We can remove `| null`.
You can confirm again.